lib/dialogs/unsynchronized/index.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+
+import AttentionIcon from '../../icons/attention';
+import Dialog from '../../dialog';
+import { closeDialog } from '../../state/ui/actions';
+import { getUnconfirmedChanges } from '../../state/simperium/functions/unconfirmed-changes';
+import exportZipArchive from '../../utils/export';
+import { noteTitleAndPreview } from '../../utils/note-utils';
+
+import type * as S from '../../state';
+import type * as T from '../../types';
+
+type OwnProps = {
+  actionDescription: string;
+  actionName: string;
+  actionSafeName: string;
+  continueAction: () => any;
```

Comment:
Given the usage above and specificity of these the repeated `action` prefixes and suffix may not be necessary.

```ts
<UnsynchronizedDialog
	description="You may lose your changes if you logout!"
	unsafeAction="Logout"
	safeAction="Safely Logout"
	continue={this.props.reallyLogout}
/>
```

`continueAction` - `action` - `finisher` - `onFinish` - `confirmAction` - `doAnyway`

the name of this one isn't as important but it seems like something else out there fits it better. if you don't like any of the others `continueAction` is fine or maybe just `continue` works.

Reply:
Resolved in https://github.com/Automattic/simplenote-electron/pull/2277/commits/bb069aed17218e442be9646f4dd46c85201bf610